CMakeLists.txt

```diff
@@ -122,11 +122,23 @@ set_target_properties(easi PROPERTIES
     CXX_STANDARD_REQUIRED ON
     CXX_EXTENSIONS OFF
 )
-target_link_libraries(easi
-    PUBLIC
-        yaml-cpp
-)
-target_include_directories(easi PUBLIC ${YAML_CPP_INCLUDE_DIR})
+
+
+if ("${yaml-cpp_VERSION}" VERSION_GREATER_EQUAL "0.8")
```

Comment:
if (yaml-cpp_VERSION VERSION_GREATER_EQUAL "0.7") better?

Reply:
I had tried it in the SeisSol repository (where we have 0.7 in the CI right now)—but apparently it complained about it not finding it; therefore I've set the minimum version to 0.8 for now (tested it on LUMI with SeisSol and 0.8; there it worked).

Reply:
ah sorry I meant the synthax, anyway does not make much difference (chatgpt was arguing that it is better ;-) )